cpp/ql/src/experimental/Security/CWE/CWE-703/FindIncorrectlyUsedExceptions.cpp

```diff
@@ -0,0 +1,9 @@
+...
+throw ("my exception!",546); // BAD
+...
+throw errorFunc("my exception!",546); // GOOD 
+...
+std::runtime_error("msg error"); // BAD
+...
+throw std::runtime_error("msg error"); // GOOD
```

Comment:
Can we have these two cases in the tests as well?

You'll need to declare just enough of `std::runtime_error` first, for example:
```
namespace std
{
	class exception {
	};

	class runtime_error : public exception {
	public:
		runtime_error(const char *msg);
	};
}
```

Reply:
it didn't work for me.
I'll try again building on your code.
Thank you.

Reply:
good day
sorry for delay.

Reply:
Thanks.